docs/component-guides/kserve.md

```diff
@@ -0,0 +1,129 @@
+# Component Guide for KServe/KFServing on AWS
+
+## Model serving over Load Balancer
+
+This tutorial shows how to setup a load balancer endpoint for serving prediction requests over an external DNS on AWS.
+
+### Background
+
+Read the [background](../deployment/add-ons/load-balancer/README.md#background) section of the load balancer guide to familiarize yourself with the requirements for creating an Application Load Balancer on AWS.
+
+### Prerequisites:
+1. To keep this guide simple, we will assume you have a Kubeflow deployment with the [AWS Load Balancer controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/) configured because you either have:
+    - A Cognito integrated deployment which is configured with [AWS Load Balancer controller by default](../deployment/cognito/README.md#30-configure-ingress) **or**
+    - You have followed the [Exposing Kubeflow over Load Balancer guide](../deployment/add-ons/load-balancer/README.md) for non-Cognito based deployment option e.g. [Vanilla](../deployment/vanilla/README.md)(uses Dex as auth provider)
+1. Based on the above pre-requisites, you have a subdomain for hosting Kubeflow. For this guide, we will assume you have `platform.example.com` for hosting Kubeflow
+1. An existing [profile namespace](https://www.kubeflow.org/docs/components/multi-tenancy/getting-started/#manual-profile-creation) for a user in Kubeflow. For this guide, we will assume you have a profile namespace named `staging`
+1. Verify the current directory is the root of the repository by running the `pwd` command. The output should be `<path/to/kubeflow-manifests>` directory
+
+
+### Customize Knative Serving domain
```

Comment:
could we number these, for some reason I wasn't able to realize that these were steps to follow until after completing the doc?